transmart-oauth/grails-app/services/org/transmart/oauth/OAuth2SyncService.groovy

```diff
@@ -59,10 +58,10 @@ class OAuth2SyncService {
                 dirty = true
 
                 if (prop == 'clientSecret') {
-                    /* Workaround to bug https://github.com/grails/grails-core/issues/10451 due to which event handlers such as beforeInsert()/beforeUpdate() are
+                    /* Workaround to bug XXX due to which event handlers such as beforeInsert()/beforeUpdate() are
```

Comment:
Why is it replaced with XXX now? Is it a mistake during rebasing? I thought that it was first set to XXX and later on Jan put a link when he created an issue.